salt/engines/slack.py

```diff
@@ -135,6 +146,25 @@ def fire(tag, msg):
                                 # Trim the ! from the front
                                 # cmdline = _text[1:].split(' ', 1)
                                 cmdline = salt.utils.shlex_split(_text[len(trigger):])
+
+                                # Remove inital salt item from list
+                                if cmdline[0] == 'salt':
```

Comment:
Right you are. I had somehow gotten it into my head that the trigger was !salt service.restart...

Would it be appropriate to expand the trigger as a single "!" is pretty generic? There could be other integrations that also operate on a short triggers, so a unique key seems important.